### DISCUSSION TITLE
blog/frontend/3-reflow-repaint

### DISCUSSION THREAD

**Discussion by giscus** [pySoo/Next-blog#64] [General]
# blog/frontend/3-reflow-repaint

어떻게 하면 웹 사이트를 더 매끄럽게 동작시킬 수 있을까요? Reflow와 Repaint에 집중해서 최적화 방법을 알아봅시다.

https://enjoydev.life/blog/frontend/3-reflow-repaint

**Comment by jjjoina** [comment]
유익한 정보 감사합니다
